cla_public/static-src/stylesheets/_forms.scss

```diff
@@ -1,21 +1,7 @@
-@import 'forms/inputs';
-@import 'forms/buttons';
 @import 'forms/field-help';
 @import 'forms/field-more-info';
-@import 'forms/field-error';
-@import 'forms/labels';
-@import 'forms/form-error';
 @import 'forms/form-group';
-@import 'forms/form-subfield';
```

Comment:
Reference point for the other comment I made

Reply:
The fact that I deleted it here indicates that I don't think it is needed, but I can't remember so will double check.

Reply:
This deletion was correct.